### PR TITLE
docs: add content to get started page

### PR DESCRIPTION
**Pull request by ollema**

### CODE REVIEW

File: sites/docs/content/get-started.md [hunk 1]
```diff
@@ -5,11 +5,45 @@ tagline: Guide
...
 
-The documentation is still a work in progress. Please refer to the [GitHub README](https://github.com/svecosystem/mode-watcher) for more information.
+You can install the project via `npm` or `pnpm`.
 
-</Callout>
+```bash
+pnpm add mode-watcher
+```
+
+<Step>Add the ModeWatcher component</Step>
```
Comment (ollema) [inline sites/docs/content/get-started.md:20]: I could not get backticks/monspace font to work on the `ModeWatcher` in this step title here. not sure if that's intended

File: sites/docs/content/get-started.md [hunk 2]
```diff
@@ -5,11 +5,45 @@ tagline: Guide
...
+
+Add the `<ModeWatcher />` component to your root `+layout.svelte` file.
+
+```svelte {2,5}#add
```
Comment (ollema) [inline sites/docs/content/get-started.md:24]: not sure about indentation here... are we supposed to indent after using `<Steps>`?

Comment (ollema) [reply sites/docs/content/get-started.md:24]: if I try to indent it's reverted by prettier it seems

Comment (huntabyte) [reply sites/docs/content/get-started.md:24]: Indentations in markdown are treated as code blocks if there is a line between the component and the plaintext!

**Review by huntabyte — APPROVED** [review]

**Pull request merged**